modules/util/portal-tools-db-upgrade-client/src/testFunctional/dependencies/check-upgrade-client-gogo-shell-connection.expect

```diff
@@ -5,7 +5,7 @@ spawn [db.upgrade.client.home]/db_upgrade_client[file.suffix.bat] -s
 set timeout 900
 
 expect {
-	"Connecting to Gogo shell." {}
+	"Connecting to Gogo shell" {}
```

Comment:
Might conflict with changes @lucasperj is making in https://liferay.atlassian.net/browse/LPS-189664